_extras/checkout.md

```diff
@@ -153,113 +138,94 @@ The lessons you may submit to are:
-
-Once you are familiar with a lesson:
-
-1. Add yourself to an upcoming discussion session on [this Etherpad]({{page.discussionpad}})
-   (or put yourself in the list at the top
-   if there isn't an upcoming discussion session for your lesson that fits your schedule).
-2. The person running your discussion session
-   will add video conferencing details to [the Etherpad]({{page.discussionpad}})
-   before the session starts.
-   At present,
-   most sessions use Google Hangouts,
-   so please make sure you have a Google account or some other login that will work with Hangouts.
-3. Please also use a headset with a built-in microphone during the session
-   rather than open-air speakers and your laptop's built-in microphone,
-   since the latter often lead to audio quality problems.
+The link to connect to the video conference is at the top of the Etherpad.
+Please be sure to use a headset with a built-in microphone during the session
+rather than open-air speakers and your laptop's built-in microphone,
+since the latter often lead to audio quality problems.
+
+To prepare for your session, pick one lesson that you plan on teaching and read the whole lesson. 
+As you are reading, make notes about the following:
+
+1. Was there anything that you found unclear in the material? Did you get stuck anywhere?  
+2. Did you feel like you would like to change the flow or present the concepts differently? If so, what and why?  
+3. What are some misconceptions that the students may have when learning this material? 
+Which bits of the lesson may be tricky and if poorly explained, may lead the students to broken mental models?  
+4. What prior knowledge do learners need to have before participating in this lesson? If you were to list any prerequisites, what would they be?  
+
+The first two questions are intended to help you become more comfortable with our collaborative lesson development methods. 
+The third and fourth question refer back to the instructor training course, and are meant to give you a chance to apply some of what you’ve learned to these specific lessons.
 
 During the session,
-you are expected to ask questions about issues you noticed while reading your chosen lesson,
-and about our operations in general.
-If the discussion leader feels that you have adequately prepared,
-she will notify the training coordinator
-and ask you to sign up for a demonstration lesson (described in the next session).
-If she feels you have not prepared,
-she may ask you to come back for another session later.
-
-### Rubric
-<a name="rubric"></a>
-
-Trainees are expected to ask questions during their discussion session
-to show that they've thought about what they're going to teach,
-and about workshops in general.
+you are expected to ask questions to show that you've thought about what you're going to teach,
+and about how our workshops are run in general.
+
 Sample questions include:
 
```

Comment:
I think we also should encourage people to ask questions about the things they were asked to think about when going through the lesson.